### PR TITLE
[No QA] fix: Fixed error with frequently used emojis not showing up

### PR DESCRIPTION
**Pull request by mananjadhav**
### Details

- When working with #7125 we changed the logic for dynamic spacing. A bug was introduced where Frequently Used Emojis stopped showing up and this PR fixes it.

### Fixed Issues

$ https://github.com/Expensify/App/issues/7074

### Tests
1. Tested frequently used emojis in the EmojiPicker

- [x] Verify that no errors appear in the JS console

### QA Steps

- [x] Verify that no errors appear in the JS console

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web

image

#### Mobile Web

Screenshot 2022-01-15 at 9 47 10 AM

#### Desktop

Screenshot 2022-01-15 at 9 50 09 AM

#### iOS

image

#### Android

Screenshot 2022-01-15 at 10 03 50 AM

### CODE REVIEW

**Comment by mananjadhav** [comment]
@stitesExpensify  @parasharrajat @rushatgabhane When working with the last EmojiPicker PR, I introduced a bug where frequently used emojis were not showing up. Adding a fix for this and I am hoping if this can be CPed to staging.

File: src/libs/EmojiUtils.js [hunk 1]
```diff
@@ -140,7 +140,7 @@ function mergeEmojisWithFrequentlyUsedEmojis(emojis, frequentlyUsedEmojis = [])
     }];
 
     allEmojis = allEmojis.concat(frequentlyUsedEmojis, emojis);
-    allEmojis = addSpacesToEmojiCategories(emojis);
+    allEmojis = addSpacesToEmojiCategories(allEmojis);
```
Comment (rushatgabhane) [inline src/libs/EmojiUtils.js:143]: return `addSpacesToEmojiCategories(allEmojis)` directly.

**Review by rushatgabhane — CHANGES_REQUESTED** [review]
Thanks for noticing and pushing the fix! 
Small change requested

File: src/libs/EmojiUtils.js (hunk 1 above)
Comment (parasharrajat) [reply src/libs/EmojiUtils.js:143]: +1

Comment (mananjadhav) [reply src/libs/EmojiUtils.js]: Updated

**Comment by mananjadhav** [comment]
PR updated

**Review by rushatgabhane — APPROVED** [review]
LGTM!
cc: @stitesExpensify

**Review by stitesExpensify — APPROVED** [review]
LGTM

**Comment by roryabraham** [comment]
Whoa, it seems like the CP label may not have worked here ... 🤔 I've never seen this so I'm going to investigate.

**Comment by roryabraham** [comment]
Oh, the CP label was added _after_ the PR was merged.

**Pull request merged**